website/docs/tutorials/basic/your_first_app/3_using_config.md

```diff
@@ -34,4 +34,37 @@ Missing mandatory value: waldo
         reference_type=Optional[Dict[Any, Any]]
         object_type=dict
 ```
+
+OmegaConf supports [interpolations](https://omegaconf.readthedocs.io/en/latest/usage.html#variable-interpolation) via [resolvers](https://omegaconf.readthedocs.io/en/latest/usage.html#custom-interpolations). Hydra supports the following resolvers by default.
```

Comment:
This is not something that belongs in the basic tutorial.
Maybe here?
https://hydra.cc/docs/next/configure_hydra/intro

Reply:
Hmm i think this is helpful for  configuring user applications too, the usage is not limited to just configuring Hydra.

Reply:
I agree but it's not basic, please move it out of the basic tutorial.

Reply:
Ok. How about creating a new tab ```Default OmegaConf Resolver``` under ( https://hydra.cc/docs/next/advanced/) ?
@omry

Reply:
It's not a big enough topic that deserves visibility there.
I think Configuring Hydra is the most appropriate category at this point.
If this grows and become much bigger it might early a slot in Advanced.